docs/contracts.rst

```diff
@@ -543,7 +543,7 @@ Fallback Function
 =================
 
 A contract can have exactly one unnamed function. This function cannot have
-arguments and cannot return anything.
+arguments, cannot return anything and has to have ``external`` visibility.
```

Comment:
I think this should have good into the "breaking PR", but it doesn't matter too much.